dev/TabView/TabView.xaml

```diff
@@ -37,7 +49,9 @@
                                 <ColumnDefinition Width="*" x:Name="RightContentColumn"/>
                             </Grid.ColumnDefinitions>
 
-                            <Border BorderBrush="{ThemeResource TabViewBorderBrush}" BorderThickness="1" Height="1" Grid.ColumnSpan="4" VerticalAlignment="Bottom"/>
+                            <Border x:Name="LeftBorder" BorderBrush="{ThemeResource TabViewBorderBrush}" BorderThickness="1" Height="1" Grid.Column="0" VerticalAlignment="Bottom"/>
```

Comment:
As mentioned in the cpp file, I think it would be better to replace the generic 'Border' with 'BottomBorderline', everywhere it applies.  Up to you.

Reply:
Done!